build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -d "node_modules/.bin" ]; then
+  echo "Be sure to run \`npm install\` before building graphiql-workspace."
+  exit 1
+fi
+
+rm -rf dist/ && mkdir -p dist/
+babel src --out-dir dist/
+echo "Bundling graphiql-workspace.js..."
+browserify -g browserify-shim -s GraphiQL dist/index.js > graphiql-workspace.js
```

Comment:
There problem here is with `-s` (standalone) option. In case of GraphiQL, it exposes `GraphiQL` react component. In our case we actually need to expose at least 2 of them: `GraphiQLWorkspace` and `AppConfig`. Not sure what would be the best way to do it: maybe export additional object in `index.js` and put these 2 component in there (let's say it;s called `graphiqlWorkspace`) then we can provide following option to browserify instead: `-s graphiqlWorkspace`